docs/gradescope.css

```diff
@@ -1,3 +1,8 @@
 .wy-side-nav-search, .wy-nav-top {
   background-color: #1b827f;
 }
+
+/* Hack around overlapping padding issue */
+.rst-versions .rst-current-version {
+  padding: 0 12px !important;
```

Comment:
In https://assets.readthedocs.org/static/css/badge_only.css this has 12px padding all around which is causing problems. The base CSS for this is in https://gradescope-autograders.readthedocs.io/en/latest/css/theme_extra.css. I'm adding !important so that we can override all other versions, which is a bit hacky and brittle but hopefully it solves the immediate problem.